public/stylesheet.css

```diff
@@ -216,11 +219,87 @@ a {
   text-decoration: none;
 }
 
+/* Modal */
+
+.popup__top {
+  top: -webkit-calc(var(--header-top-offset) + 65px);
+  top: -moz-calc(var(--header-top-offset) + 65px);
+  top: calc(var(--header-top-offset) + 65px);
+  right: var(--side-offset);
+}
+@media (max-width: 860px) {
+  .popup__top {
+    top: -webkit-calc(var(--header-top-offset) + 65px);
+    top: -moz-calc(var(--header-top-offset) + 65px);
+    top: calc(var(--header-top-offset) + 65px);
+  }
+}
+@media (max-width: 640px) {
+  .popup__top {
+    top: -webkit-calc(var(--header-top-offset) + 65px);
+    top: -moz-calc(var(--header-top-offset) + 65px);
+    top: calc(var(--header-top-offset) + 65px);
+  }
+}
+
+.popup-trigger {
+  display: inline-block;
+}
+
+.popup_modal {
+  background-color: #fff;
+  position: absolute;
+  border: 0.1rem solid var(--color-button-border);
```

Comment:
It seems that the top corners of the modal are not rendered, at least for me:

#### Firefox - Light

![image](https://user-images.githubusercontent.com/23049315/139758267-2c901311-2570-48eb-85b4-11b5c0f89c61.png)

#### Chrome - Dark

![image](https://user-images.githubusercontent.com/23049315/139758338-4bc3320a-cbaf-4688-b752-45f35e5812e0.png)

#### Chrome - Light

![image](https://user-images.githubusercontent.com/23049315/139758445-349ff028-2003-4c5a-b1f0-fbdcf5c00f18.png)

It seems fixed with a slightly wider border. Could you double check it?

```suggestion
  border: 0.13rem solid var(--color-button-border);
```

Reply:
I don't know why that is, it shows just fine for me in Chrome and acceptably fine in Firefox